skimage/measure/_moments_analytical.py

```diff
@@ -0,0 +1,157 @@
+                                        + cz*(cy*m[0, 1, 0] - m[0, 2, 0])
+                                        + m[0, 2, 1])
+            moments_central[0, 3, 0] = (2*cy**2*m[0, 1, 0]
+                                        - 3*cy*m[0, 2, 0]
+                                        + m[0, 3, 0])
+            moments_central[1, 0, 2] = (-cx*m[0, 0, 2]
+                                        + 2*cz*(cx*m[0, 0, 1] - m[1, 0, 1])
+                                        + m[1, 0, 2])
+            moments_central[1, 1, 1] = (-cx*m[0, 1, 1]
+                                        + cy*(cx*m[0, 0, 1] - m[1, 0, 1])
+                                        + cz*(cx*m[0, 1, 0] - m[1, 1, 0])
+                                        + m[1, 1, 1])
+            moments_central[1, 2, 0] = (-cx*m[0, 2, 0]
+                                        - 2*cy*(-cx*m[0, 1, 0] + m[1, 1, 0])
+                                        + m[1, 2, 0])
+            moments_central[2, 0, 1] = (cx**2*m[0, 0, 1]
+                                        - 2*cx*m[1, 0, 1]
+                                        + cz*(cx*m[1, 0, 0] - m[2, 0, 0])
+                                        + m[2, 0, 1])
+            moments_central[2, 1, 0] = (cx**2*m[0, 1, 0]
+                                        - 2*cx*m[1, 1, 0]
+                                        + cy*(cx*m[1, 0, 0] - m[2, 0, 0])
+                                        + m[2, 1, 0])
+            moments_central[3, 0, 0] = (2*cx**2*m[1, 0, 0]
+                                        - 3*cx*m[2, 0, 0]
+                                        + m[3, 0, 0])
+
+    return moments_central.astype(float_dtype, copy=False)
+
+
+def moments_raw_to_central(moments_raw):
+    ndim = moments_raw.ndim
+    order = moments_raw.shape[0] - 1
+    if ndim in [2, 3] and order < 4:
+        return _moments_raw_to_central_fast(moments_raw)
+
+    moments_central = np.zeros_like(moments_raw)
+    m = moments_raw
+    # centers as computed in centroid above
+    centers = tuple(m[tuple(np.eye(ndim, dtype=int))] / m[(0,)*ndim])
+
+    if ndim == 2:
+        # This is the general 2D formula from
+        # https://en.wikipedia.org/wiki/Image_moment#Central_moments
+        for p in range(order + 1):
+            for q in range(order + 1):
+                if p + q > order:
+                    continue
+                for i in range(p + 1):
+                    term1 = math.comb(p, i)
+                    term1 *= (-centers[0]) ** (p - i)
+                    for j in range(q + 1):
+                        term2 = math.comb(q, j)
+                        term2 *= (-centers[1]) ** (q - j)
+                        moments_central[p, q] += term1*term2*m[i, j]
+        return moments_central
+
+    # The nested loops below are an n-dimensional extension of the 2D formula
+    # given at https://en.wikipedia.org/wiki/Image_moment#Central_moments
+
+    # iterate overall [0, order] (inclusive) on each axis
```

Comment:
```suggestion
    # iterate over all [0, order] (inclusive) on each axis
```

Reply:
Sorry to jump in here, you all are doing much better at helping maintain and improve scikit-image than I am. 

May I suggest that these stylistic nits at the end of a review be simply "applied" instead of suggested?

This would enable the merge to just go through.

The back and forth takes a lot of core developer time, and I think it would help move projects forward if these smaller nits were made in a more assertive fashion.

^_^ I guess it is more of how you want to collaborate with others, but I see these as "net positive"

Reply:
Agreed, Mark! Per our contributor guide we do that for newcomers, and check with core devs first. FWIW you all can push corrections to all my branches! I can always revert them if I don't like them 😎

Reply:
Thank you for making the suggestions, @mkcor. From my end it is also fine to just push these kind of edits directly in the future.

Reply:
> May I suggest that these stylistic nits at the end of a review be simply "applied" instead of suggested?

Agreed as well. I would have committed these and merged the PR today (Monday), had there been no activity in the meantime. I tend to leave things 'pending' (not blocking) over the weekend.